.github/workflows/_deploy_kafka_test.yml

```diff
@@ -0,0 +1,198 @@
+name: Setup Workload Identity 
+
+on:
+  workflow_call:
+    inputs:
+      skr-client-akv-endpoint:
+        description: "AKV endpoint for key release"
+        required: true
+        type: string
+      skr-client-maa-endpoint:
+        description: "Attestation endpoint"
+        required: true
+        default: "sharedeus2.eus2.test.attest.azure.net"
```

Comment:
Make this a Github Secret.

Reply:
I think secrets are meant for sensitive credentials and stuff. I think we can make this as a repo scoped variable.